### PR TITLE
Refactored LargeValueFormatter

### PR DESCRIPTION
**Pull request by MaximPestryakov**

### CODE REVIEW

File: MPChartLib/src/main/java/com/github/mikephil/charting/formatter/LargeValueFormatter.java [hunk 1]
```diff
@@ -68,8 +68,12 @@ public void setAppendix(String appendix) {
      *
      * @param suff new suffix
```
Comment (RobertZagorski) [inline MPChartLib/src/main/java/com/github/mikephil/charting/formatter/LargeValueFormatter.java]: Please correct this text.

**Comment by RobertZagorski** [comment]
@almic This seems to me as ready to be merged. As this is a small change and can be quickly reviewed and possibly merged.

**Comment by almic** [comment]
Do not `@` mention me unless I've already participated in a PR.

Anyway this looks fine.

**Comment by almic** [comment]
To clarify, I am already notified about new comments and PR discussions, @ mentioning me just spams me with a new email.

**Pull request merged**